hue/hue/base/hue-mysql-dc.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: hue-mysql
+  annotations:
+    template.alpha.openshift.io/wait-for-ready: "true"
+spec:
+  strategy:
+    type: Recreate
+  triggers:
+    - type: ImageChange
+      imageChangeParams:
+        automatic: true
+        containerNames:
+          - mysql
+        from:
+          kind: ImageStreamTag
```

Comment:
Is this an IS coming from `openshift` namespace? Do we need more control over version?

Reply:
If we change this to the `Deployment` this whole section will go away.

~Also it's already controlled in the kustomization (including the namespace).~ You've found that. :)

Reply:
Changed to `Deployment`